src/scene/light.js

```diff
@@ -463,7 +463,7 @@ class Light {
             this.uploadAreaLightLUTs();
         }
 
-        this._shape = value;
+        this._shape = value ? value : LIGHTSHAPE_PUNCTUAL;
```

Comment:
Is this to guard against value being `null` or `undefined`? If so, won't it be undesirable to call `uploadAreaLightLUTs` above? Where are 'bad' values being passed to this function anyway?

Reply:
now using a sanitizedValue - I'm not sure where the bad values are coming from - I had assumed the _shape property was set once to the property default on construction - and left alone if not used - I can investigate further.

Reply:
used default param for sanitizing value - also removed incorrect assignment in initializeComponentData()